documentation/asciidoc/topics/proc_running_server.adoc

```diff
@@ -8,14 +8,14 @@ This starter first attempts to locate the server from the `hotrod-client.propert
 +
 [source,text,options=nowrap]
 ----
-include::config_examples/hotrod-client.properties[]
+infinispan.client.hotrod.server_list=127.0.0.1:11222
```

Comment:
the file `documentation/asciidoc/topics/config_examples/hotrod-client.properties` can be removed.

Reply:
thanks, removed.